packages/salesforcedx-vscode-apex-debugger/README.md

```diff
@@ -62,6 +62,13 @@ To start a debugging session, from the configuration dropdown menu at the top of
 
 While a debugging session is in progress, any synchronous activity that runs a line of code with a breakpoint causes execution to halt at the breakpoint. While execution is paused, you can inspect the call stack and see the current values of your variables. You can also step through your code, using the Debug actions pane that appears at the top of the editor while a debugging session is in progress, and watch those values change. You can debug up to two threads at a time. For more information, see [Debugging](https://code.visualstudio.com/docs/editor/debugging) in the Visual Studio Code docs.
 
+## Set Exception Breakpoints
+To make the Apex Debugger halt execution when an exception is thrown during a debugging session, set breakpoints on exceptions. When an exception breakpoint is hit, the debugger pauses on the line of code that caused the exception to be thrown.
+
+To set an exception breakpoint, in VS Code, run **Apex Debug: Configure Exceptions**. The list of available exceptions includes the [exceptions in the `System` namespace](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_classes_exception_methods.htm) and the Apex classes in your project that extend `Exception`. Select an exception from the list, and then select **Always break**. 
```

Comment:
Are you obviously in "VS Code"? If there's no ambiguity regarding where you are or should be, I would remove ", in VS Code, "